src/main/java/core/basesyntax/FigureSupplier.java

```
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final double MIN_INT = 1;
+    private static final double MAX_INT = 10;
+    private static final double DEFAULT_INT = 10;
```

The constants MIN_INT, MAX_INT and DEFAULT_INT are declared but their names are misleading (they are doubles representing dimensions). Consider renaming them to `MIN_DIMENSION`, `MAX_DIMENSION`, and `DEFAULT_RADIUS` to better reflect their purpose. This aligns with the checklist requirement to use well-named constants for magic numbers. See lines 7-9.